draft-ietf-taps-impl.md

```diff
@@ -887,29 +887,82 @@ Connection Object:
 
 ## SCTP
 
-To support sender-side stream schedulers (which are implemented on the sender side),
-a receiver-side Transport System should
-always support message interleaving {{!RFC8260}}.
-
-SCTP messages can be very large. To allow the reception of large messages in pieces, a "partial flag" can be
-used to inform a (native SCTP) receiving application that a
-message is incomplete. After receiving the "partial flag", this application would know that the next receive calls will only
-deliver remaining parts of the same message (i.e., no messages or partial messages will arrive on other
-streams until the message is complete) (see Section 8.1.20 in {{!RFC6458}}). The "partial flag" can therefore
-facilitate the implementation of the receiver buffer in the receiving application, at the cost of limiting
-multiplexing and temporarily creating head-of-line blocking delay at the receiver.
-
-When a Transport System transfers a Message, it seems natural to map the Message object to SCTP messages in order
-to support properties such as "Ordered" or "Lifetime" (which maps onto partially reliable delivery with
-a SCTP_PR_SCTP_TTL policy {{!RFC6458}}). However, since multiplexing of
-Connections onto SCTP streams may happen, and would be hidden from the application, the
-Transport System requires a per-stream receiver buffer anyway, so this potential benefit is lost
-and the "partial flag" becomes unnecessary for the system.
-
-The problem of long messages either requiring large receiver-side buffers or getting in the way of
-multiplexing is addressed by message interleaving {{!RFC8260}},
-which is yet another reason why a receivers-side transport system supporting SCTP should
-implement this mechanism.
+\[EDITOR'S NOTE: The paragraph below is placed here because, currently, only the SCTP text follows this convention. However, the suggestion is to move this paragraph up, to appear directly under "Specific Transport Protocol Considerations", and also update TCP and UDP to follow this style.]
```

Comment:
Can we have this be an issue rather that a note in the markdown? Not a huge deal either way.

I do like the format!

Reply:
If we're good with the format (waiting for an ACK from @britram), it's more efficient to just leave the editor comment for now and remove it with the follow-up PR (to change UDP and TCP in the same way), which I'll then do fast.

Reply:
.... and thanks for looking at this so quickly btw!

Reply:
woops, forgot @abrunstrom  ... adding her too.  Quick answers would be appreciated, so I can continue this method early next week if folks agree.

Reply:
Done